templates/plugins/generic/pdfJsViewer/templates/display.tpl

```diff
@@ -28,12 +28,12 @@
 	{load_stylesheet context="frontend" stylesheets=$stylesheets}
 	{load_script context="frontend" scripts=$scripts}
 </head>
-<body class="pkp_page_{$requestedPage|escape} pkp_op_{$requestedOp|escape}">
+<body class="pkp_op_{$requestedOp|escape}">
```

Comment:
I would rather leave here the class `pkp_page_{$requestedPage|escape}`. I can imagine situations when users want to use this specifier.

Reply:
On the second thought, what about using the default pages for HTML and PDf galley pages for the sake of support? @sssoz, what to do you think? In this case, we need to add styling for classes from a default theme (label for a screen reader and a return button).

Reply:
yeah, i don’t think it’s necessary to have custom HTML / PDF galley styles actually - they are very neutral anyway! In order to have those, we only need the `pkp_page_{$requestedPage|escape}` class right?

Reply:
They both represent the part of a router; it's page and associated operation (op). Each operation has attached handler (controller), e.g., ArticleHandler class, that represents a page and contains methods that correspond to operations, like view, download. 
Anyway, galleys have 2 additional parameters in URL that correspond to article id and galley id.

Reply:
Thanks for the clarifications -- fixing this now!